src/lsptoolshost/autoInsert/onAutoInsert.ts

```diff
@@ -142,9 +151,18 @@ async function applyAutoInsertEdit(
     }
 }
 
-function getFormattingOptions(): FormattingOptions {
-    const editorConfig = vscode.workspace.getConfiguration('editor');
-    const tabSize = editorConfig.get<number>('tabSize') ?? 4;
-    const insertSpaces = editorConfig.get<boolean>('insertSpaces') ?? true;
+function getFormattingOptions(document: vscode.TextDocument): FormattingOptions {
+    const editorConfig = vscode.workspace.getConfiguration('editor', document);
```

Comment:
This being called editorConfig was a bit confusing but I got passed it. Doesn't help that I code review from the bottom up...

Reply:
haha true its a bit overloaded.  I can think of a different name